src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -3432,14 +3569,15 @@ def test_fs_discontinuous_and_inc_error():
         ast = fpapi.parse(code, ignore_comments=False)
         with pytest.raises(ParseError) as excinfo:
             _ = DynKernMetadata(ast, name="testkern_qr_type")
-        assert ("It does not make sense for a field on a discontinuous "
-                "space (" + fspace + ") to have a 'gh_inc' access"
-                in str(excinfo.value))
+        assert ("In the LFRic API allowed accesses for a field on a "
```

Comment:
Comma after "API" and no need for parentheses around the function space if we quote it.

Reply:
Changed.